examples/remote-view/index.mjs

```diff
@@ -37,8 +42,9 @@ const stdlib = loadStdlib(process.env);
       keepGoing: async () => {
         const kg = count < 5;
         const ctcAddr = await ctcAlice.getContractAddress();
-        const x = await contract.callStatic["callView"](ctcAddr);
-        console.log(`Compare: `, x.toNumber(), count);
+        const f = (stdlib.connector == 'CFX') ? contract : contract.callStatic;
```

Comment:
Perhaps this means that we should change `cfxers` to support `callStatic`?

Reply:
I'll make a ticket for that